tests/bazel_run_tests.sh

```diff
@@ -39,7 +39,20 @@ function test_outdated_no_external_runfiles() {
   expect_log "junit:junit \[4.12"
 }
 
-TESTS=("test_duplicate_version_warning" "test_duplicate_version_warning_same_version" "test_outdated" "test_outdated_no_external_runfiles")
+test_xdg_cache_home() {
+  readonly cachedir=/tmp/${test}-cache
+  XDG_CACHE_HOME=$cachedir bazel run @unsafe_shared_cache//:pin &>> "$TEST_LOG"
```

Comment:
This line is failing the macOS CI build (the [log is here](https://buildkite.com/bazel/rules-jvm-external/builds/2410#588b4a7d-98b3-4de9-a4b1-7c5e5f2b17a1/77-147)). If you've got access to a Mac, could you please make sure that this test passes?

Reply:
Ack! I'll check things out today. Thanks!